core/tests/revert-test/tests/revert-and-restart-en.test.ts

```diff
@@ -359,10 +398,11 @@ describe('Block reverting test', function () {
         mainNode = await MainNode.spawn(mainLogs, enableConsensus, true);
 
         console.log('Wait for the external node to detect reorg and terminate');
-        await extNode.waitForExit();
+        await utils.sleep(3);
 
         console.log('Restart external node and wait for it to revert.');
         extNode = await ExtNode.spawn(extLogs, enableConsensus);
+        await utils.sleep(30);
```

Comment:
why do we need to sleep here?

Reply:
Although within `ExtNode.spawn` we wait for the node to be responsive, without this sleep the following L1 txn has a chance of failing. Maybe connected to the time needed to revert?